src/sundials/sundials_adaptcontroller.c

```diff
@@ -93,9 +97,9 @@ SUNAdaptController_Type SUNAdaptController_GetType(SUNAdaptController C)
  * Optional functions in the 'ops' structure
  * ----------------------------------------------------------------- */
 
-int SUNAdaptController_Destroy(SUNAdaptController C)
+SUNErrCode SUNAdaptController_Destroy(SUNAdaptController C)
```

Comment:
Should we update this to
```suggestion
SUNErrCode SUNAdaptController_Destroy(SUNAdaptController* C)
```

Reply:
Yes, but I think in a different PR.